src/main/java/apoc/coll/Coll.java

```diff
@@ -768,4 +768,16 @@ public long occurrences(@Name("coll") List<Object> coll, @Name("item") Object it
 
         return combinations;
     }
+
+    @UserFunction
+    @Description("apoc.coll.different(values) - returns true if values are different")
+    public boolean different(@Name("values") List<Object> values) {
+        for(int i=0; i<values.size()-1;i++){
```

Comment:
always use foreach
this one is just `return new HashSet(values).size() == values.size()`